indra/newview/llaccountingcostmanager.h

```diff
@@ -70,7 +70,7 @@ class LLAccountingCostManager : public LLSingleton<LLAccountingCostManager>
 	//a fetch has been instigated.
 	uuid_set_t mPendingObjectQuota;
 
-    void accountingCostCoro(std::string url, eSelectionType selectionType, const LLHandle<LLAccountingCostObserver> observerHandle);
+    static void accountingCostCoro(std::string url, eSelectionType selectionType, const LLHandle<LLAccountingCostObserver> observerHandle);
```

Comment:
Is this from another ticket?

Reply:
Yes, it's a separate commit.

Coroutines routinely outlive classes they we spawned from so they crash on shutdown trying to access 'this' once woken up after a suspend (postAndSuspend in this case). Turning coro into static and 'this' into 'self' makes things a bit easier to track in my opinion.